libraries/chain/include/graphene/chain/market_object.hpp

```diff
@@ -214,6 +215,12 @@ typedef multi_index_container<
             const_mem_fun< call_order_object, asset_id_type, &call_order_object::debt_type>
          >
       >,
+      ordered_unique< tag<by_account_and_id>,
```

Comment:
Hm. We could avoid this additional index if we paginate on `asset_id`.
I think it's very unlikely that a new `call_order_object` appears while someone is cycling through the list, especially because only very few accounts will have more than one page of call orders.
@abitmore ?

Reply:
I agree it's good to reuse `by_account`.

Reply:
ok, changed at https://github.com/bitshares/bitshares-core/pull/1749/commits/976b56bbe3fac494490ef57277e11674c1a8e597